### ISSUE TITLE
Warosu Compatibility

### ISSUE THREAD

**Issue by randoomdude** [Fiddlekins/SpookyX#19]
Are there no plans to make the extension work with Warosu archive? Would make going through a lot more convinient, there aren't any other methods of expanding images on the site, and just adding the domain to included sites in spookyx doesn't work.

**Comment by Fiddlekins** [comment]
Sorry, I have no plans to adapt it for warosu at this point.

The first step to getting it functional is to add `// @require https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js` to the metadata since warosu doesn't have that on the page by default.

After that I think nothing happens because warosu doesn't label page elements with the same ids and classes, so spookyx isn't finding anything to process. Addressing that would require more time than I'm interested in spending I'm afraid.
